internal/commands/image/list.go

```diff
@@ -177,6 +179,16 @@ func getImageListColumns(options imageListOption) []imageListColumn {
 		imageListColumn{"APP NAME", func(p pkg) string {
 			return p.bundle.Name
 		}},
+		imageListColumn{"CREATED", func(p pkg) string {
+			payload, err := packager.CustomPayload(p.bundle.Bundle)
+			if err != nil {
+				return ""
+			}
+			if createdPayload, ok := payload.(packager.CustomPayloadCreated); ok {
+				return units.HumanDuration(time.Now().UTC().Sub(createdPayload.CreatedTime())) + " ago"
+			}
+			return ""
```

Comment:
I wonder if it should be empty `""` or `"N/A"` ? @thaJeztah  WDYT?

Reply:
Ideally, would use a go template, which would allow setting `N/A` in the template (separating presentation from the value); see https://github.com/docker/cli/pull/2107

Reply:
Yep right, we're fixing it in a follow-up, thanks @thaJeztah 😸